testScript/parts/XMLConfig/projectName-selenium/src/test/java/xxxxxx/yyyyyy/zzzzzz/selenium/error/ErrorTest.java

```diff
@@ -5,21 +5,26 @@
 
 import java.time.Duration;
 
+import org.junit.After;
```

Comment:
- [ ] I don't think any modifications are necessary to the ErrorTest on the XMLConfig side either.

Reply:
bd060d3
Corrected as noted.